### ISSUE TITLE
[Feature request] Add global `text-*` classes that will explicitly apply to all breakpoints

### ISSUE THREAD

**Issue by AlexandreBonneau** [vuetifyjs/vuetify#2176]
Currently, if you want to center a text for all breakpoints, you must use the `text-xs-center` class.
This works great but it relies on the *implicit* knowledge that `text-xs-center` will not only applies to `xs` sizes, but to *xs-and-up*.

In order to make it more explicit for the user, I would like to see an alias of `text-xs-center` created so that a user could use the CSS class `text-center` (or `text-all-center`) if he wants the alignment to apply to all breakpoints.

A note in the [documentation](https://vuetifyjs.com/layout/alignment) could be added to explains how the `text-*-center` rules override each other (for instance if a user use `<p class="text-center text-md-right">`, then the text would be centered for all breakpoints except for the `md` one where it would be aligned on the right).

To sum up, I think anything *implicit* is bad.
**Explicit** FTW 👍 

*Of course, this is not only relevant for `center` but for other alignments too (ie. `right`, `left`, etc.)*

**Comment by johnleider** [comment]
@Phlow2001 is exactly right.  This is something that bootstrap does well and I mimicked. Thank you for the request, but I think this can be solved by better documentation on those classes. Adding a note to do such thing.
